bittytax/conv/parsers/ftx.py

```diff
@@ -0,0 +1,182 @@
+                                             fee_quantity=row_dict['fee'],
+                                             wallet=WALLET)
+
+
+def parse_ftx_funding_crypto(data_row, _parser, **kwargs):
+    row_dict = data_row.row_dict
+    data_row.timestamp = DataParser.parse_timestamp(row_dict['time'])
+
+    payment = Decimal(row_dict['payment'])
+    if payment > 0:
+        # Getting paid by other side
+        data_row.t_record = TransactionOutRecord(TransactionOutRecord.TYPE_TRADE,
+                                                 data_row.timestamp,
+                                                 buy_asset='USD',   # No non-USD future markers at the moment
+                                                 buy_quantity=payment,
+                                                 sell_asset=row_dict['future'],
+                                                 sell_quantity=0,
+                                                 wallet=WALLET,
+                                                 note='Funding fee')
+    else:
+        # Paying to the other side
+        data_row.t_record = TransactionOutRecord(TransactionOutRecord.TYPE_TRADE,
+                                                 data_row.timestamp,
+                                                 buy_asset=row_dict['future'],
+                                                 buy_quantity=0,
+                                                 sell_asset='USD',
+                                                 sell_quantity=abs(payment),
+                                                 wallet=WALLET,
+                                                 note='Funding fee')
+
+
+def parse_ftx_lending_crypto(data_row, _parser, **kwargs):
+    row_dict = data_row.row_dict
+    data_row.timestamp = DataParser.parse_timestamp(row_dict['time'])
+    data_row.t_record = TransactionOutRecord(TransactionOutRecord.TYPE_INTEREST,
+                                             data_row.timestamp,
+                                             buy_asset=row_dict['coin'],
+                                             buy_quantity=row_dict['proceeds'],
+                                             fee_asset='USD',
+                                             fee_quantity=row_dict['feeUsd'],
+                                             wallet=WALLET)
+
+
+DataParser(DataParser.TYPE_EXCHANGE,
+           'FTX Deposits',
+           ['id', 'time', 'coin', 'size', 'status', 'additionalInfo', 'txid', '_delete'],
+           worksheet_name='FTX D',
+           row_handler=parse_ftx_deposits_crypto)
+
+DataParser(DataParser.TYPE_EXCHANGE,
+           'FTX Withdrawals',
+           ['time', 'coin', 'size', 'address', 'status', 'txid', 'fee', 'id'],
+           worksheet_name='FTX W',
+           row_handler=parse_ftx_withdrawals_crypto)
+
+DataParser(DataParser.TYPE_EXCHANGE,
+           'FTX Trades',
+           ['id', 'time', 'market', 'side', 'type', 'size', 'price', 'total', 'fee', 'feeCurrency'],
```

Comment:
FTX has updated the headers to:
`"ID","Time","Market","Side","Order Type","Size","Price","Total","Fee","Fee Currency"`